batchflow/batch.py

```diff
@@ -493,22 +494,73 @@ def apply_transform(self, ix, func, *args, src=None, dst=None, p=None, use_self=
 
             for item in range(len(batch)):
                 self.dst[item] = func(self.src[item], *args, **kwargs)
+
+        If `src` is a list with two or more elements, `dst` should be list or
+        tuple of the same lenght.
+
+        Examples
+        --------
+
+        ::
+
+            apply_transform(make_masks_fn, src='images', dst='masks')
+            apply_transform(apply_mask, src=('images', 'masks'), dst='images', use_self=True)
+            apply_transform_all(rotate, src=['images', 'masks'], dst=['images', 'masks'], p=.2)
         """
+        to_act = bool(p is None or np.random.binomial(1, p))
+
+        if isinstance(src, list) and len(src) > 1 and isinstance(dst, (list, tuple)) and len(src) == len(dst):
```

Comment:
No need to expect `dst` to be a tuple here either.
If `src` is a list, let `dst` be a list too.

Reply:
Done